src/libraries/System.Security.Cryptography/tests/MD5Tests.cs

```diff
@@ -148,5 +154,12 @@ public async Task MD5_Rfc1321_7_AsStream_Async()
         {
             await VerifyRepeatingAsync("1234567890", 8, "57edf4a22be3c955ac49da2e2107b67a");
         }
+
+        [Fact]
+        public void Sha512_HashSizes()
```

Comment:
```suggestion
        public void MD5_HashSizes()
```

Or rename the whole passel of them:

```suggestion
        public void VerifyHashSizes()
```